components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <section class="providers">
```

Comment:
```suggestion
  <section class="providers-list">
```

to follow the naming classes with the file name

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/3020/commits/f51b5acdeb431d727230f946a5cd49632abed534